far/FarEng.hlf.m4

```diff
@@ -4024,31 +4027,35 @@ files will be listed on a single stripe.
     The following column types are supported:
 
     N[M[D],O,R[F],N] - file name
-                 where: M - show selection marks
-                            where: D - dynamic selection marks;
-                        O - show names without paths
-                            (intended mainly for plugins);
-                        R - right align names that do not fit in column
-                            where: F - right align all names;
-                        N - do not show extensions in name column;
-                 These modifiers can be used in combination,
-                 for example NMR
+                 where: M - ^<wrap>show selection marks where: D
+- dynamic selection marks;
+                        O - ^<wrap>show names without paths (intended
+mainly for plugins);
+                        R - ^<wrap>right align names that do not fit in
+column where: F - right align all names;
+                        N - ^<wrap>do not show extensions in name
+column;
+                 ^<wrap>These modifiers can be used in combination, for
+example NMR
 
     X[R]       - file extension
-                 where: R - right align file extension;
+                 where: R - ^<wrap>right align file extension;
 
     S[C,T,F,E] - file size
     P[C,T,F,E] - allocation file size
     G[C,T,F,E] - size of file streams
-                 where: C - format file size;
-                        T - use 1000 instead of 1024 as a divider;
-                        F - show file sizes similar to Windows
-                            Explorer (i.e. 999 bytes will be
-                            displayed as 999 and 1000 bytes will
-                            be displayed as 0.97 K);
-                        E - economic mode, no space between file
-                            size and suffix will be shown
-                            (i.e. 0.97K);
+                 where: C - ^<wrap>group digits using the symbol from
+Windows settings;
+                        T - ^<wrap>use decimal units instead of binary,
+i.e., to calculate kilobytes, the size will be divided by 1000 instead
+of by 1024; in this mode unit symbol is shown in lower case, e.g., #k#,
+#m#, #g# instead of #K#, #M#, #G#;
+                        F - ^<wrap>show size like Windows Explorer, as
+a decimal fraction with no more than three digits before decimal point,
+e.g., 999 bytes will be shown as #999#, while 1000 bytes as #1.00 k#;
```

Comment:
This is actual behavior; probably not what was intended. I could not get `0.97 K` from the formatting algorithm though. I can fix the implementation if you like, but I would throw away all the math there and use a table of hard-coded thresholds to choose units.

Reply:
- Since "T" is not on by default, let's change this to "while 1024 bytes as 1.00 K"
- "show size like Windows Explorer" is useless in this particular example- these days Explorer shows 0 bytes as 0 K and 1 byte as 1 K.